packages/conventional-changelog-conventionalcommits/parser-opts.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+module.exports = function (config) {
+  return {
+    headerPattern: /^(\w*)(?:\((.*)\))?: (.*)$/,
+    headerCorrespondence: [
+      `type`,
+      `scope`,
+      `subject`
+    ],
+    noteKeywords: [`BREAKING CHANGE`],
```

Comment:
Should this also be configurable?

Reply:
maybe eventually? I think I'd like to implement a first pass that addresses the most common feature requests that I've seen:

* the ability to more directly control the URLs to commits, ranges, and issues in the CHANGELOG generated.
* the ability to configure which types of commits show up in the CHANGELOG.

I expect we'll want to make more options configurable as we start to work our way through existing feature requests in the backlog of issues.